tests/test_plugin.py

```diff
@@ -1529,3 +1529,26 @@ def test_nested_namespace_consistency(descriptor: PluginDescriptor) -> None:
                 f"{desc.name} ({desc.module}.{desc.qualname})" for desc in result if desc.namespace != part
             )
             pytest.fail(f"Namespace name {descriptor.namespace!r} has conflicts with function name: {conflicts}")
+
+
+@pytest.mark.parametrize(
+    "descriptor",
+    [descriptor for descriptor in plugins() if NamespacePlugin in descriptor.cls.__bases__],
```

Comment:
```suggestion
    [descriptor for descriptor in plugins() if issubclass(descriptor.cls, NamespacePlugin)],
```
Wouldn't `issubclass` work?